xtask/src/metrics.rs

```diff
@@ -26,7 +26,7 @@ impl flags::Metrics {
         }
         {
             let _d = pushd("./target/rustc-perf")?;
-            cmd!("git reset --hard 1d9288b0da7febf2599917da1b57dc241a1af033").run()?;
+            cmd!("git reset --hard c52ee623e231e7690a93be88d943016968c1036b").run()?;
```

Comment:
We should add this hash to JSON we publish to the metrics repo, so that it's easy to see when we update the test suite.